src/Laravel/config/api-platform.php

```diff
@@ -32,18 +32,18 @@
 
     'formats' => [
         'jsonld' => ['application/ld+json'],
-        'jsonapi' => ['application/vnd.api+json'],
+        //'jsonapi' => ['application/vnd.api+json'],
     ],
 
     'patch_formats' => [
         'json' => ['application/merge-patch+json'],
     ],
 
     'docs_formats' => [
-        'jsonopenapi' => ['application/vnd.openapi+json'],
-        'json' => ['application/json'],
         'jsonld' => ['application/ld+json'],
+        'jsonopenapi' => ['application/vnd.openapi+json'],
         'html' => ['text/html'],
+        //'json' => ['application/json'],
```

Comment:
let's remove this one completely no? I'm think that this is useful only if you want to reach `/docs.json`

Reply:
Done. I added YAML Open API support instead.